packages/p2p/src/components/orders/order-details/order-details.jsx

```diff
@@ -89,7 +90,7 @@ const OrderDetails = ({ order_information, chat_info }) => {
                             {is_completed_order && (
                                 <div className='order-details__wrapper-message'>{labels.result_string}</div>
                             )}
-                            {(is_pending_order || is_buyer_confirmed_order) && (
+                            {!has_timer_expired && (is_pending_order || is_buyer_confirmed_order) && (
```

Comment:
Should we also add this condition to where we show the footer?

Reply:
The conditions aren't the same for showing the footer, e.g. we definitely want to show the footer if the timer has expired in some cases.